picture-marker-symbols/src/main/java/com/esri/arcgisruntime/sample/picturemarkersymbols/MainActivity.java

```diff
@@ -35,168 +36,185 @@
+    final PictureMarkerSymbol campsiteSymbol = new PictureMarkerSymbol(
+        "http://sampleserver6.arcgisonline.com/arcgis/rest/services/Recreation/FeatureServer/0/images/e82f744ebb069bb35b234b3fea46deae");
+    //Optionally set the size, if not set the image will be auto sized based on its size in pixels, its appearance would
+    //then differ across devices with different resolutions.
+    campsiteSymbol.setHeight(18);
+    campsiteSymbol.setWidth(18);
+    campsiteSymbol.loadAsync();
+    //[DocRef: END]
+    campsiteSymbol.addDoneLoadingListener(new Runnable() {
+      @Override
+      public void run() {
+        //Once the symbol has loaded, add a new graphic to the graphic overlay
+        Point campsitePoint = new Point(-223560, 6552021, SpatialReferences.getWebMercator());
+        Graphic campsiteGraphic = new Graphic(campsitePoint, campsiteSymbol);
+        graphicOverlay.getGraphics().add(campsiteGraphic);
+      }
+    });
+
+    //[DocRef: Name=Picture Marker Symbol Drawable-android, Category=Fundamentals, Topic=Symbols and Renderers]
+    //Create a picture marker symbol from an app resource
+    BitmapDrawable pinStarBlueDrawable = (BitmapDrawable) ContextCompat.getDrawable(this, R.drawable.pin_star_blue);
+    final PictureMarkerSymbol pinStarBlueSymbol = new PictureMarkerSymbol(pinStarBlueDrawable);
+    //Optionally set the size, if not set the image will be auto sized based on its size in pixels, its appearance would
+    //then differ across devices with different resolutions.
+    pinStarBlueSymbol.setHeight(40);
+    pinStarBlueSymbol.setWidth(40);
+    //Optionally set the offset, to align the base of the symbol aligns with the point geometry
+    pinStarBlueSymbol.setOffsetY(11); //The image used for the symbol has a transparent buffer around it, so the offset is not simply height/2
+    pinStarBlueSymbol.loadAsync();
+    //[DocRef: END]
+    pinStarBlueSymbol.addDoneLoadingListener(new Runnable() {
+      @Override
+      public void run() {
+        //add a new graphic with the same location as the initial viewpoint
+        Point pinStarBluePoint = new Point(-226773, 6550477, SpatialReferences.getWebMercator());
+        Graphic pinStarBlueGraphic = new Graphic(pinStarBluePoint, pinStarBlueSymbol);
+        graphicOverlay.getGraphics().add(pinStarBlueGraphic);
+      }
+    });
+
+    //Check sample has access to external storage
+    if (saveResourceToExternalStorage()) {
+
+      //[DocRef: Name=Picture Marker Symbol File-android, Category=Fundamentals, Topic=Symbols and Renderers]
+      //Create a picture marker symbol from a file on disk
+      BitmapDrawable pinBlankOrangeDrawable = (BitmapDrawable) Drawable.createFromPath(mPinBlankOrangeFilePath);
+      final PictureMarkerSymbol pinBlankOrangeSymbol = new PictureMarkerSymbol(pinBlankOrangeDrawable);
+      //Optionally set the size, if not set the image will be auto sized based on its size in pixels, its appearance would
```

Comment:
@willcrick You should split this comment after the comma to the next line comment.